static/src/javascripts/projects/common/modules/experiments/tests/mt-sticky-burger.js

```diff
@@ -0,0 +1,224 @@
+define([
+    'fastdom',
+    'common/utils/$',
+    'common/utils/_',
+    'common/utils/config',
+    'common/utils/detect',
+    'common/utils/mediator'
+], function (
+    fastdom,
+    $,
+    _,
+    config,
+    detect,
+    mediator
+) {
+    return function () {
+        this.id = 'MtStickyBurger';
+        this.start = '2015-04-21';
+        this.expiry = '2015-05-21';
+        this.author = 'Zofia Korcz';
+        this.description = 'Sticky top banner with navigation - variant 1. with the burger icon';
+        this.audience = 0.02;
+        this.audienceOffset = 0.03;
+        this.successMeasure = '';
+        this.audienceCriteria = '1% of US edition';
+        this.dataLinkNames = '';
+        this.idealOutcome = '';
+
+        this.canRun = function () {
+            var isIE = detect.getUserAgent.browser === 'MSIE' || detect.getUserAgent === 'IE 11',
+                isUS = config.page.edition === 'US';
+
+            return !isIE && isUS;
+        };
+
+        function scrollDirection(scrollY, config) {
+            if (scrollY > config.prevScroll) {
+                config.direction = 'down';
+            } else if (scrollY < config.prevScroll) {
+                config.direction = 'up';
+            }
+            config.prevScroll = scrollY;
+
+            return config.direction;
+        }
+
+        function showNavigationMobile(scrollY, config) {
+            if (scrollDirection(scrollY, config) === 'up') {
+                config.$stickyNavigation.show();
+            } else {
+                config.$stickyNavigation.hide();
+            }
+        }
+
+        function showNavigation(scrollY, config) {
+            if (scrollDirection(scrollY, config) === 'up') {
+                config.$navigationScroll.css('display', 'block');
+                config.$navigationGreySection.css('border-top', '36px solid #00456e');
+            } else {
+                config.$navigationScroll.css('display', 'none');
+            }
+        }
+
+        function updatePosition(config) {
+            fastdom.write(function () {
+                if (window.scrollY < config.scrollThreshold) {
```

Comment:
You are not testing it on IE but just FYI [scrollY](https://developer.mozilla.org/en-US/docs/Web/API/Window/scrollY) is not working on IE.